Feat (Advanced Analytics): Chart interaction FAQ (#2946)
Kong/developer.konghq.com: app/advanced-analytics/explorer.md

        * **Export as CSV**: If you prefer to analyze your data using other tools, you can download the current view as a CSV file, making it portable and ready for further analysis elsewhere.   
  - q: What interactions are available when working with charts to investigate my data?
    a: |
       Charts in {{site.konnect_short_name}} Analytics include interactive options to help you investigate data:
       - **Hover** over a chart to see a tooltip with exact values.
       - **Left-click** to pin the tooltip. This lets you scroll through long lists or drag the tooltip to a different spot.
       - **Click and drag** across the chart to highlight a time range. When you release, choose one of the following actions: 
            - **View requests**: Opens **Analytics > API Requests** with a filtered list of matching calls.
            - **Explore**: Opens Explorer with the same filters and highlighted time window, so you can add group-bys and extra filters.
            - **Zoom**: Focuses the chart on the selected time range.
       

>        From the Chart menu:

Reviewer: ```suggestion
       From the chart settings, you can do the following:
```
I was looking for something called "Chart", so this confused me :)

>        - To open Explorer with the chart’s current filters and time range, without highlighting first, select **Explore**.

Reviewer: ```suggestion
       - Open Explorer with the chart’s current filters and time range, by selecting **Explore**.
```

>        - To download the chart view, select **Export CSV**. The file includes the current filters and time window.

Reviewer: ```suggestion
       - Download the chart view by selecting **Export CSV**. The file includes the current filters and time window.
```
